### ISSUE TITLE
Clown Raid

### ISSUE THREAD

**Issue by TripleThreeT** [Tslat/Advent-Of-Ascension#3384]
Most people are in agreement that the Celeve Dimension is the worst in the game and needs a drastic overhaul or should be removed entirely. Well, what if instead of eliminating the mobs from the game, you instead implement them into a Raid similar to the pillagers?! You could give certain mobs new functions such as having clown cars, balloons, and water-shooting flowers, but they will only ever appear as raid mobs. The only exception is a rare Clown mob that gives the "Clown Omen" effect. There's also the possibility of making these raids interdimensional. For example, you can activate the Raid in a Dystopia village. The rare clown will also spawn in every dimension.
The end of the raid will feature Gyro as a final boss.

**Comment by EatingSilencer** [comment]
Problem with this is all bosses are getting moved to Nowhere, so having a boss in a raid will not work.

**Comment by TripleThreeT** [comment]
Yes, you're right. I guess the item used to spawn the boss can be dropped in the raid somehow.
